CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 ### Fixed
 
 - Fixed nextBoolean method for XoShiRo256StarStar generators.
+- Fixed wrong order of seed and count inputs in *Random Color* node.
```

Comment:
We don't add CHANGELOG entries for changes in the same subversion.

Reply:
Okay, thanks for the info.